.github/actions/collect-test-data/action.yml

```diff
@@ -25,7 +25,7 @@ runs:
   using: 'composite'
   steps:
     - name: Collect test data
-      uses: getsentry/action-collect-test-data@c0a2fd3f79d3867e2c8b486ccb1c4bf925477742 # v0.3.0
+      uses: getsentry/action-collect-test-data@ac0b9ecb5dc7437c113051e4e6a70012ff02c6dc # 0.4.1
```

Comment:
unrelated -- looking at the output it looks like this action calls `setup-python` which clobbers the python state of the workflow.  it shouldn't need to do this and should be fine with just stdlib and `/usr/bin/python3` ?